src/inputs/input/input.jsx

```diff
@@ -29,46 +50,133 @@ const propTypes = {
      * Deprecated prop. Please use `disable` instead.
      */
     disabled: PropTypes.bool,
+    /**
+     * Indicates that the input has an error.
+     */
     error: PropTypes.oneOfType([
         PropTypes.bool,
         PropTypes.string,
     ]),
+    /**
+     * An input can take on the size of its container.
+     */
     fluid: PropTypes.bool,
+    /**
+     * Tells the component whether to be in guide or no guide mode.
+     */
     guide: PropTypes.bool,
+    /**
+     * Optional icon to display inside the input.
+     */
     icon: PropTypes.oneOfType([
         PropTypes.shape({}),
         PropTypes.string,
     ]),
+    /**
+     * The id is used for the a label\'s for.
```

Comment:
Maybe:
```
Assign an element ID to the Input.
```

Reply:
Please check this commit https://github.com/saddlebackdev/react-cm-ui/pull/353/commits/8429b8ffd987325a6ae504db3aa2341b60de60ef

Reply:
Looks good.  Resolving.